anomalib/models/patchcore/model.py

```diff
@@ -163,12 +165,13 @@ def forward(self, input_tensor: Tensor) -> Union[torch.Tensor, Tuple[torch.Tenso
             embedding = self.tiler.untile(embedding)
 
         embedding = self.reshape_embedding(embedding)
+        feature_map_shape = embedding.shape[-2:]
```

Comment:
The feature map has to be computed before reshaping the embedding for this to work correctly. So this line needs to be placed above line 167.

Reply:
right!